definitions/ext-sap_idoc/golden_metrics.yml

```diff
@@ -0,0 +1,24 @@
+avgProcTime:
+  title: Avg Processing Time
+  unit: SECONDS
```

Comment:
We have a tab that is causing a [linter failure](https://github.com/newrelic/entity-definitions/pull/403#pullrequestreview-851385872) either one this line above or [here](https://github.com/newrelic/entity-definitions/pull/403/files#diff-e7039db7097f4f2ce20c0ef4f23ca5c9f7575a561d5a6e226c622cbbe14e14d7R3). Could we please replace it as a space?

Reply:
I couldn't find the tab there. Just retyped the whole line to clean up.

Reply:
Summited the changed file.